hospitals.Rmd

```diff
@@ -0,0 +1,116 @@
+---
+title: "Hospitals"
+author: "Marcela"
+date: "2/25/2021"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo=FALSE, warning=FALSE)
+```
+
+```{r load_libraries}
+library(dplyr)
+library(leaflet)
+library(sf)
+library(htmlwidgets)
+library(rgdal)
```

Comment:
is rgdal needed, I don't see it being used? its a big package if not :)

Reply:
Line deleted! It wasn't necessary :)